test/test_unit_keystore_adapter.py

```diff
@@ -37,3 +35,10 @@ def test_is_keyring_accessible_return_true_when_get_password(self, mock_keyring)
     @patch.object(keyring, "get_password", side_effect=keyring.errors.KeyringError)
     def test_is_keyring_accessible_returns_false_on_keyring_error(self, mock_keyring):
         self.assertEquals(False, KeystoreAdapter.is_keyring_accessible())
+
+    @patch.object(util_functions, "configure_env_var_with_keyring")
+    def test_env_variables_set_on_keyring_adapter_import(self, mock_utils):
+        import importlib
+        import conjur.wrapper.keystore_adapter
```

Comment:
why are these here are not at the top of the file?

Reply:
could move 'importlib' the top but I think that `import conjur.wrapper.keystore_adapter` should be here so the test will be understandable. I will add comment